motulator/common/model/__init__.py

```diff
@@ -1,14 +1,26 @@
 """Common functions and classes for continuous-time system models."""
 from motulator.common.model._simulation import (
```

Comment:
A minor thing, but these imported items (and those in __all__) could be in the alphabetical order. Then, the order is consistent and we don't need to think it when, e.g., adding something.